app/components/common/FilterSelect.tsx

```diff
@@ -90,7 +102,19 @@ export default function FilterSelect(
                 {options.slice(maxOptionsToDisplayCollapsed).map((option, key) => (
                   <FormControlLabel
                     key={key}
-                    control={<Checkbox name={option.name} checked={filterIsChecked(option)} onChange={selectFilter} />}
+                    control={
+                      <Checkbox
+                        name={option.name}
+                        checked={filterIsChecked(option)}
+                        onChange={selectFilter}
+                        sx={{
+                          color: 'common.white',
```

Comment:
Move the style into a new variable and use it for all the checkboxes in the FilterSelect.